1-getting-started/lessons/4-connect-internet/translations/README.bn.md

```diff
@@ -0,0 +1,438 @@
+
+> 💁 প্রাচীনতম টেলিমেট্রি ডিভাইসগুলি ফ্রান্সে ১৮৭৪ সালে উদ্ভাবিত হয়েছিল এবং Mont Blanc থেকে প্যারিসে রিয়েল-টাইম আবহাওয়া এবং তুষারের গভীরতা প্রেরণ করেছিল। এটি বাস্তবিক তারগুলি ব্যবহার করত কারণ ওয়্যারলেস প্রযুক্তিগুলি তখন ছিল  না।
+
+চলুন পাঠ 1 থেকে স্মার্ট থার্মোস্টেটের উদাহরণটি দেখি।
+
+![একাধিক রুম সেন্সর ব্যবহার করে ইন্টারনেটে সংযুক্ত একটি থার্মোস্ট্যাট](../../../../images/telemetry.png)
+
+টেলিমেট্রি সংগ্রহের জন্য থার্মোস্ট্যাট এর তাপমাত্রাভিত্তিক সেন্সর রয়েছে। এটিতে একটি তাপমাত্রাভিত্তিক সেন্সর বিল্টইন থাকে এবং ওয়ারলেস  প্রোটোকল যেমন [ব্লুটুথ লো এনার্জি](https://wikipedia.org/wiki/Bluetooth_Low_Energy) (BLE) এর মাধ্যমে একাধিক তাপমাত্রাভিত্তিক সেন্সর এর সাথে  সংযুক্ত হতে পারে।
+
+An example of the telemetry data it would send could be:
+
+| নাম | মান | বর্ণনা |
+| ---- | ----- | ----------- |
+| `thermostat_temperature` | 18°C | থার্মোস্ট্যাট এর বিল্ট-ইন তাপামাত্রাভিত্তিক সেন্সর দ্বারা তাপমাত্রা পরিমাপ করা। |
+| `livingroom_temperature` | 19°C |  দূরবর্তী তাপমাত্রাভিত্তিক সেন্সর(remote temperature sensor) দ্বারা তাপমাত্রা পরিমাপ করা হয় যেটিকে `livingroom` নামকরণ করা হয়েছে যাতে এটি যেই রুমে আছে সেই রুমকে শনাক্ত করতে পারে। |
+| `bedroom_temperature` | 21°C | দূরবর্তী তাপমাত্রাভিত্তিক সেন্সর(remote temperature sensor) দ্বারা তাপমাত্রা  পরিমাপ করা হয় যেটিকে ` bedroom ` নামকরণ করা হয়েছে যাতে এটি যেই রুমে আছে সেই রুমকে শনাক্ত করতে পারে। |
+
+ক্লাউড সেবা এই টেলিমেট্রি ডেটা ব্যবহার করে তাপকে নিয়ন্ত্রণ করতে কী আদেশ পাঠাবে তার সিদ্ধান্ত নিতে পারে।
+
+### টেলেমেট্রি আইওটি ডিভাইসে প্রেরণ
+
+নাইটলাইটটিকে ইন্টারনেটের মাধ্যমে নিয়ন্ত্রণ করার পরবর্তী অংশটি হলো লাইট লেভেল টেলেমেট্রি MQTT- ব্রোকারের টেলেমেট্রি টপিকে পাঠানো।
+
+#### কাজ
+
+লাইট লেভেল টেলেমেট্রি MQTT- ব্রোকারে পাঠানো।
+
+ডেটা JSON হিসাবে এনকোড করে পাঠানো হয় – JSON হলো JavaScript Object Notation এর সংক্ষিপ্ত রূপ, যা কী/ভ্যালু পেয়ার ব্যবহার করে ডেটাকে এনকোডেড টেক্সট এ রূপান্তরের জন্য একটি স্ট্যান্ডার্ড ।
+
+✅ JSON সম্পর্কে জ়েনে না থাকলে তা এই [JSON.org documentation](https://www.json.org/) থেকে শিখতে পারবো।
+
+ডিভাইস থেকে MQTT-ব্রোকারের কাছে টেলিমেট্রি প্রেরণের জন্য নীচের পদক্ষেপটি অনুসরণ করিঃ
+
+* [আরডুইনো Wio টার্মিনাল](wio-terminal-telemetry.bn.md)
+* [সিংগেল বোর্ড কম্পিউটার - রাস্পবেরি পাই/ভার্চুয়াল আইওটি ডিভাইস](single-board-computer-telemetry.bn.md)
+
+### MQTT ব্রোকার হতে টেলেমেট্রি গ্রহণ
+
+টেলিমেট্রি পাঠানোর কোন অর্থ নেই যদি অন্য প্রান্তে এটি  listen-করার মতো কিছু না থাকে। লাইট লেভেল টেলিমেট্রিটির ডেটা প্রক্রিয়া করার জন্য এটিকে কিছু লিসেনিং(listening) এর প্রয়োজ়নীয়তা আছে । এই 'সার্ভার' কোডটি সেই ধরণের কোড যা বৃহত্তর আইওটি অ্যাপ্লিকেশনের অংশ হিসাবে একটি ক্লাউড সেবাতে স্থাপন করবো, তবে এখানে স্থানীয়ভাবে এই কোডটি লোকাল কম্পিউটারে চালাতে পারবো (বা আমাদের Pi-তে সরাসরি কোডিং করতে পারবো) । সার্ভার কোডে একটি পাইথন অ্যাপ থাকে যা MQTT-র মাধ্যমে লাইটে লেভেলগুলোর টেলেমেট্রি বার্তা listen করতে পারে । এই পাঠের পরবর্তীতে, এটিতে কমান্ডের মেসেজ রিপ্লেতে পাঠাবো যাতে নির্দেশনা থাকবে যে এলইডি অন না অফ হবে।
+
+✅ কিছু গবেষনা করিঃ MQTT বার্তাগুলোর কী হবে যদি কোনো listener না থাকে?
```

Comment:
✅ কিছু গবেষনা করিঃ MQTT বার্তাগুলোর কী হবে যদি কোনো গ্রাহক না থাকে?

Reply:
@Iftu119 should I use **গ্রাহক/গ্রহণ** instead of **Listen/Listening** here or should I left it as it is?